authors/apps/articles/tests/test_article_readtime.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import status
+from authors.apps.articles.tests import base_class
+from ..utils.utils import get_article_read_time
+
+
+class TestArticleReadTime(base_class.BaseTest):
+    """Class tests calculation of read time of an article"""
+
+    def setUp(self):
+        self.body = "Hey you, how are you doing?"
```

Comment:
Could this body be pulled to the test data and clearly labelled, for example with `one_min_read_text` so it is clear what you are trying to achieve?

Reply:
Just to recap on what you are trying to get me to achieve, would you like me to get this from a created article and supply the body?

Reply:
No, this very same text, but moved to the test data file.
Someone might for example change it in future not knowing that you had already measured that it took a minute to read.
Does this make sense?

Reply:
Makes sense